pkg/server/image_pull.go

```diff
@@ -403,3 +410,16 @@ func newTransport() *http.Transport {
 		ExpectContinueTimeout: 5 * time.Second,
 	}
 }
+
+// addEncryptedImagesPullOpts adds the necessary pull options to a list of
```

Comment:
name is wrong in comment

Reply:
s/adds the necessary pull options to a list of pull options if enabled/returns the list of configured encrypted image pull handler options/ ??

Reply:
should be encrypted... not addEncrypted...

Reply:
does not add to a list it returns the list